src/endpoint/s3/s3_bucket_policy_utils.js

```diff
@@ -136,6 +137,12 @@ async function _is_object_tag_fit(req, predicate, value) {
     dbg.log1('bucket_policy: object tag fit?', value, tag, res);
     return res;
 }
+async function _is_object_version_fit(req, predicate, value) {
+    const version_id = req.query.versionId;
+    const res = predicate(version_id, value);
+    dbg.log1('Condition statement for version-id, res :', version_id, res);
```

Comment:
can you add value to the log, so we will be able to compare version ids to see why the condition failed/accepted

Reply:
done